infrastructure/modules/dns/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "metadata" {
+  type = object({
+    solution    = string
```

Comment:
Hva er solution? Og hva er instance? Og hvorfor er dette 1 variabel og ikke 3?

Reply:
Kan vi ihvertfall omdoepe solution og instance til noe som gir mening? name og suffix kanskje?

Reply:
instance er ikke en suffix, brukt mange steder og er bla. en del av MS sin navnestandard: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-naming

Reply:
Fair - og synes "workload" eller "application" var et mye bedre navn enn "solution"

Reply:
vi kan bytte til navn fra solution

Reply:
kan også utvide metadata med suffix = "${name}-${instance}-${environment}"

Reply:
> kan også utvide metadata med suffix = "${name}-${instance}-${environment}"

Ikke helt sikker paa om jeg skjoente hva du mente her?

Reply:
suffixen for de fleste ressursene er ganske lik. Vi kan utvide metadata objektet med feltet suffix som inneholder en statisk string som alle ressurser skal bruke når man spesifiserer navn, så slipper man at man gjør "sb-${name}-${instance}-${environment}" og heller bare "sb-${suffix}"

Reply:
Ah - ja, det er ikke dumt. Dog - den boer kanskje ikke hete `suffix`. Jeg foeler at du da skal ha noe mer enn `sb-` forran. `sb-${defaultName}`?